README.md

```diff
@@ -1,2 +1,64 @@
-# hw01
-Repository for homework 01
+## Alexan Martin-Eichner: A Brief (Auto)Biography
+
+(you can also call me Sasha)
+
+---
+
+# I USE THEY/THEM/THEIR PRONOUNS
+
+![this is me][1]
+
+### Wait who are you and why haven't i seen you in class
+
+Great question! I have several chronic health conditions which I am still learning to functionally cope with. I also have frequent medical appointments, which are often very urgent and very difficult to reschedule. I usually love to overshare, but i feel like a list of my diagnoses would be considered a little inappropriate here (?).
+
+### Life story, up to now
+
+Raised in a wealthy WASP household with two parents and two older siblings just outside of Washington, DC. I knew that UChicago was my top choice for undergrad because of its
+
+* reputation for theoretical rigor
+* strong social science division
+* unofficial slogan "where fun goes to die"
+
+Attending UChicago would turn out to be one of the **worst**, or at least most harrowing, decisions of my life. I'm honestly thankful that me and my friends all made it through alive (not everyone can say the same). I got one BA in Mathematics and another BA with honors in Sociology.
+
+---
+
+_Fun fact_: i am obsessed with wacky mixed methods! Check out this directed graph illustrating the system of conditional entailments constructed from a qualitative-comparative coincidence analysis i used in my sociology BA thesis!
+
+![the semantics are kinda complicated][2]
+
+---
+
+##### (life story cont.)
+
+I wanted a research-related MA so I could have a fancy stamp on my resume making it slightly easier to get industry research jobs. I decided to settle for UChicago, despite my experiences in undergrad, because
+
+1. i already live in hp
+2. i already know how the faculty and administration like to operate
+3. i got a 75% scholarship
+
+And im taking this class because I think some of the most promising mixed-methods in social science today are computational (and it looks good on a resume)!
+
+### Personality
+
+**ENFJ**
+
+![NATAL BIRTH CHART IN THE WESTERN TROPICAL SYSTEM][3]
+
+### Identity (Politics)
+
+upper-middle class white trans nonbinary buddhist neurodivergent disabled anarchist communist vegan bisexual polyamorous sub
+
+### Things i like to like on social media
+
+* [books and print comics][4]
+* [music][5]
+* [movies][6]
+
+[1]: figures/me.JPG "this is me"
+[2]: figures/thesisgraph.png "i can explain i promise"
+[3]: figures/birthchart.png "yes i have a stellum in aquarius AND capricorn, its a wild ride"
+[4]: https://www.goodreads.com/user/show/20850992-alexan-martin-eichner "goodreads"
+[5]: https://rateyourmusic.com/~alexanme "rateyourmusic"
+[6]: https://letterboxd.com/alexanme/ "letterboxd"
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* This is excellent code
* I enjoyed reading your biography, especially that you voluntarily decided to go to UChicago for undergrad... me too!
* I don't really have any criticisms